cmd/loop/loopout.go

```diff
@@ -63,6 +63,20 @@ var loopOutCommand = cli.Command{
 				"satoshis, if not specified, a default max " +
 				"fee will be used",
 		},
+		cli.Float64Flag{
+			Name: "max_sweep_fee_portion",
+			Usage: "the max on-chain miner fee used for sweeping " +
```

Comment:
I wonder if we should use percent here, but still keep it as a float flag? So it would be 5% by default. To make it a bit more in line with some of the autoloop options that also use percent. What do you think, @carlaKC?

Reply:
Capping the miner fee means that you leave it subject to a potential timeout sweep

Reply:
> I wonder if we should use percent here, but still keep it as a float flag?
Initially I did use percent, concerned it might cause confusion. TBH I don't know which one is better, let me know and I'll change it.

> Capping the miner fee means that you leave it subject to a potential timeout sweep
Indeed, I think the idea here is to leave it to users to decide how much fee they want to pay, instead of the 100x we used before.

Reply:
Cool, percent it is.